README.md

```diff
@@ -109,6 +109,14 @@ Specifically:
 
     Commit messages should be verbose by default, consisting of a short subject line (50 chars max), a blank line and detailed explanatory text as separate paragraph(s); unless the title alone is self-explanatory (like "Corrected typo in main.cpp") then a single title line is sufficient. Commit messages should be helpful to people reading your code in the future, so explain the reasoning for your decisions. Further explanation [here](http://chris.beams.io/posts/git-commit/).
 
+### Code Style
+
+Please use `black` to format code automatically before opening a PR:
```

Comment:
Not sure how much effort it would involve, but it might be nice to use a GH action to make sure this step has happened. Smth like https://github.com/marketplace/actions/python-black-check.

Reply:
Yes, I agree. I have https://github.com/dashpay/sentinel/pull/110/commits/0049757f35a5be0c2c06a96bbb838383ff9ccac5 which has the check, not sure if that GH Action you referenced also creates a commit for non-conforming code or how I would feel about a bot creating a commit on top of a PR. I think it just does a check though, which is what we have set up already.

I think it's a bit crazy that the defaults in that Action deviate from the defaults in black (line length 81 vs 88). The reason I like black is b/c it's very low-config, just use defaults and don't tweak anything, so for that reason would prefer to use our own (seems like less work too).

Reply:
Ah yeah, no worries, at first I thought you were suggesting making another commit to auto-fix formatting if necessary.